packer/build_ovf.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash -e
```

Comment:
```ovf``` is a standard packaging format . it's not only used for Virtualbox/Vagrant , but also for VMWare( but I believe you will have another script to build the cached/templated RackHD Vmware image )
 I think it's more proper to rename this script to build_base_vagrant_box.sh or something more clear ?